### PR TITLE
Update Pillow component to use Image instead of SVG

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Refactor `Pillow` component to render and image instead of SVG.

## Justify why they are needed

The first iteration of the site will use images for pillows.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1045** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1045" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1045?utm_source=stack-comment).

**Review by gustaveen — APPROVED** [review]
Cool! 🙌 Just so we are aware, this will likely be some vector or 3d shape even in the end but let's care about that later :)

File: apps/store/src/components/ProductPage/PurchaseForm/CircledHSuperscript.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import styled from '@emotion/styled'
+
+export const CircledHSuperscript = () => {
+  return <Superscript>Ⓗ</Superscript>
+}
+
+const Superscript = styled.sup({
```
Comment (guilhermespopolin) [inline apps/store/src/components/ProductPage/PurchaseForm/CircledHSuperscript.tsx:7]: Didn't know about this `<sub>` tag. 😮 
You never stop learn

**Review by guilhermespopolin — APPROVED** [review]

File: apps/store/src/components/ProductPage/PurchaseForm/CircledHSuperscript.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/ProductPage/PurchaseForm/CircledHSuperscript.tsx:7]: HTML is pretty powerful - no matter what @mehdielket thinks ;)

**Review by robinandeer — COMMENTED** [review]
@gustaveen yes, but we agreed that for the first release it will be an image I believe :)

**Pull request merged**